server/api/batch_inventory.py

```diff
@@ -483,15 +487,22 @@ def download_batch_inventory_ballot_manifest(
     csv_io = io.StringIO()
     ballot_manifest = csv.writer(csv_io)
 
-    ballot_manifest.writerow(["Batch Name", "Number of Ballots"])
+    ballot_manifest.writerow(["Container", "Batch Name", "Number of Ballots"])
```

Comment:
is there a reason we call this `container` in the manifest, but `counting group` in our code? Might this be confusing? It makes me wonder about what a counting group really is.

Reply:
Yeah, we are just reusing the existing Container column that Arlo already supports, since this is just an experiment. I didn't want to add a new column to the manifest yet. It is a little confusing, yes